assets/js/test-net.js

```diff
@@ -37,29 +44,35 @@ function rippleTestNetCredentials(url, altnet_name) {
   sequence.html('')
   loader.css('display', 'inline')
 
+  // generate the test wallet
+  const test_wallet = xrpl.Wallet.generate();
 
   //call the alt-net and get key generations
   $.ajax({
     url: url,
     type: 'POST',
+    contentType: "application/json; charset=utf-8",
+    data: JSON.stringify({
+      destination: test_wallet.address,
+    }),
     dataType: 'json',
     success: function(data) {
       //hide the loader and show results
       loader.hide();
       credentials.hide().html('<h2>Your '+altnet_name+' Credentials</h2>').fadeIn('fast')
       address.hide().html('<h3>Address</h3> ' +
-        data.account.address).fadeIn('fast')
+        test_wallet.address).fadeIn('fast')
       secret.hide().html('<h3>Secret</h3> ' +
-        data.account.secret).fadeIn('fast')
+      test_wallet.seed).fadeIn('fast')
       balance.hide().html('<h3>Balance</h3> ' +
-        Number(data.balance).toLocaleString('en') + ' XRP').fadeIn('fast')
+        Number(data.amount).toLocaleString('en') + ' XRP').fadeIn('fast')
```

Comment:
I believe the `amount` is how much the Faucet sent, whereas `balance` is how much they have now; if you're adding funds to an existing account, your actual "balance" is of course the sum of how much you had before plus how much you received.

Currently it doesn't matter because we are still funding fresh accounts, but if we build the extension that I have in mind to let users save their testnet credentials in browser storage for reuse, it would be inaccurate to call the "amount" and "balance" the same. The accurate "Balance" in that case, which the user probably would like to know, is of course, their previous balance _plus_ the amount the faucet sent.

For now, could you just leave a comment for later reminding us to adjust this if we allow wallet reuse? Also, if you wanted to merge the changes from #1535 into this PR I wouldn't object.

Reply:
yeah so currently the blocker is that the faucet code doesn't send balance info unless it's creating an account, the balance is updated through `wait_for_seq,` I just added that as a temporary balance until the  ` wait_for_seq` returns a result. but I will make a note.